### DISCUSSION TITLE
Making Hallucinations More Confusing

### DISCUSSION THREAD

**Discussion by Minukelsus** [FakeFishGames/Barotrauma#7117] [Ideas and suggestions]
Idk if this is a good idea but imagine you're swimming in the sea. You suddenly get attacked by a sea life but it doesn't do any damage to you. But you still panic, firing and shouting for help thinking it's a real enemy when it's just a hallucination.

Same could be done on a sub where someone hallucinates an AI attacking them that has someone's existing name on the sub harassing or attacking them. 

This is just so there's more panic and realistic factor like the sudden fires or broken junction boxes that you can't tell if they're real or not.

Maybe throw in hallucinations where you see an enemy on the radar but no one else can see it.

**Comment by StruggleDuck** [comment]
This sounds like a great idea, would help increase the severity of psychosis, as right now it is just an inconvenience rather than an actual obstacle.

**Comment by ghost** [comment]
Do you mean psychosis or hallucinations btw? They're 2 different things.

**Comment by Thirty-Percent** [comment]
I had suggested something like this as well last year (for psychosis) 

https://github.com/Regalis11/Barotrauma/issues/4378#issue-749200112
